### ISSUE TITLE
Update to v0.62.0 got this error when use it.

### ISSUE THREAD

**Issue by wind13** [atom/vim-mode#1016]
Uncaught TypeError: undefined is not a function
/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:679
Hide Stack Trace
TypeError: undefined is not a function
  at getChangesSinceCheckpoint (/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:679:6)
  at VimState.module.exports.VimState.deactivateInsertMode (/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:456:15)
  at VimState.module.exports.VimState.activateNormalMode (/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:392:6)
  at atom-text-editor.registerCommands.activate-normal-mode (/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:66:35)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:519:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:354:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:178:20)

**Comment by wind13** [comment]
And uninstall then reinstall the v0.65.0 got this error:

Failed to activate the vim-mode package
undefined is not a function
Hide Stack Trace
TypeError: undefined is not a function
    at getChangesSinceCheckpoint (/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:1290:26)
    at VimState.module.exports.VimState.deactivateInsertMode (/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:1026:17)
    at VimState.module.exports.VimState.activateNormalMode (/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:939:12)
    at new VimState (/Users/samliu/.atom/packages/vim-mode/lib/vim-state.coffee:76:14)
    at /Users/samliu/.atom/packages/vim-mode/lib/vim-mode.coffee:28:22
    at Workspace.module.exports.Workspace.observeTextEditors (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:266:9)
    at Object.module.exports.activate (/Users/samliu/.atom/packages/vim-mode/lib/vim-mode.coffee:22:43)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:222:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:203:30
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)

**Comment by AlexNPavel** [comment]
Update your Atom version. I was just having this issue as the atom-editor-bin package in the Arch Linux AUR is still on version 1.6.2, due to library compatibility issues. I installed the atom-editor package instead which builds atom locally and is on version 1.7.3 and that fixed the issue.

TL;DR: Update to Atom Editor 1.7.3

**Comment by wind13** [comment]
Yes, it solved. Thanks!
